package.json

```diff
@@ -42,31 +42,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "prettier . --write",
+    "lint": "eslint src/"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
       "eslint:recommended",
       "plugin:react/recommended",
       "plugin:@typescript-eslint/recommended",
-      "plugin:jsx-a11y/strict"
+      "plugin:jsx-a11y/strict",
+      "plugin:prettier/recommended"
     ],
     "plugins": [
       "react-hooks",
       "@typescript-eslint",
       "jsx-a11y"
     ],
     "rules": {
+      "no-case-declarations": "off",
```

Comment:
This looks like a good rule, why is it being switched off? Is it because it's happening in many places? If it's only happening in a handful of places, it is worth fixing it. If it happens in many places, then switching it to at least a warning would be preferrable.

Reply:
This was already in the rules from UoC, not one I added. I removed this line and fixed the single occurrence